### DISCUSSION TITLE
Enabling SSL (https)

### DISCUSSION THREAD

**Discussion by whateverx28** [code4recovery/12-step-meeting-list#1464] [General]
I thought SSL (https) was enabled by default on our site, but apparently not. In WP > Meetings > Settings, I see this message "If you enable SSL (https), your users will be able to search near their location." Can you tell me how to do that? The meetings plugin works perfectly. Thank you for that.

**Comment by tim-rohrer** [comment]
Hi @whateverx28,

The process for enabling SSL on a site varies a little by the provider being used. This is something you'll want to contact your provider to determine. 

TSML doesn't have anything to do with how to set up SSL.

Good luck!

**Reply by whateverx28** [reply]
Tim, thank you. I thought the provider provided SSL and it seems to me the site is SSL enabled, but I'll check again with provider. I saw this message in Meetings, which is why I ask you all. Thanks again

**Reply by whateverx28** [reply]
Fixed with free SSL from wpengine, the provided. Thanks again!
